src/services/verify/fragments.tsx

```diff
@@ -1,23 +1,24 @@
-import { isValid, VerificationFragment } from "@govtechsg/oa-verify";
-const getFirstFragmentFor = (fragments: VerificationFragment[], name: string) =>
+import { isValid, VerificationFragment, AllVerificationFragment } from "@govtechsg/oa-verify";
+
+const getFirstFragmentFor = (fragments: AllVerificationFragment[], name: string) =>
   fragments.filter((status) => status.name === name)[0];
 
 const revokeFragmentName = "OpenAttestationEthereumDocumentStoreRevoked";
-export const getNotRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment<any>[] =>
+export const getNotRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment[] =>
   fragments.filter((status) => status.name !== revokeFragmentName && status.status !== "SKIPPED");
-export const getRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment<any>[] =>
+export const getRevokeFragment = (fragments: VerificationFragment[]): VerificationFragment[] =>
   fragments.filter((status) => status.name === revokeFragmentName && status.status !== "SKIPPED");
 
 // this function check if the reason of the error is that the document store or token registry is invalid
-export const addressInvalid = (fragments: VerificationFragment[]): boolean => {
+export const addressInvalid = (fragments: AllVerificationFragment[]): boolean => {
   const documentStoreIssuedFragment = getFirstFragmentFor(fragments, "OpenAttestationEthereumDocumentStoreIssued");
   const tokenRegistryMintedFragment = getFirstFragmentFor(fragments, "OpenAttestationEthereumTokenRegistryMinted");
   // 2 is the error code used by oa-verify in case of invalid address
   return documentStoreIssuedFragment?.reason?.code === 2 || tokenRegistryMintedFragment?.reason?.code === 2;
 };
 
 // this function check if the reason of the error is that the document store or token has not been issued
-export const certificateNotIssued = (fragments: VerificationFragment[]): boolean => {
+export const certificateNotIssued = (fragments: AllVerificationFragment[]): boolean => {
```

Comment:
im searching for the use for this function, doesnt seem to be in use at all
this one and `addressInvalid`. 

could someone double check? maybe it was refactored out before and now these are no longer needed.

Reply:
ok have remove the useless functions. i ran testes and made sure all working fine with out the code.